test.py

```diff
@@ -2096,6 +2096,49 @@ def test_variants(self):
         self.assertNotIn("Variant", game.headers)
 
 
+class CraftyTestCase(unittest.TestCase):
+
+    def setUp(self):
+        try:
+            self.engine = chess.xboard.popen_engine("crafty")
+        except OSError:
+            self.skipTest("need crafty")
+
+        self.engine.xboard()
+        self.engine.send_line("log off")
+
+    def tearDown(self):
+        self.engine.quit()
+
+    def testSt(self):
```

Comment:
The other tests methods are currently named in `snake_case` (except setUp/tearDown, where there is no choice).